modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/display/context/builder/CPSpecificationOptionsFacetDisplayContextBuilder.java

```diff
@@ -132,11 +131,9 @@ public void renderRequest(RenderRequest renderRequest) {
 
 		_facet = facet;
 
-		Optional<String[]> parameterValuesOptional =
+		parameterValues(
```

Comment:
@julschong `portletSharedSearchResponse.getParameterValues()` is possible to return null, why do you ignore the null check?

Reply:
@dantewang my thought on this was that `_parameterValues` was null to start so having it equal to null doesn't do any different. but now that there is no null check, if it was not null and this method gets called again, it will overwrite the value to become null again now. I should keep this null check.